display_list/skia/dl_sk_canvas.h

```diff
@@ -10,8 +10,12 @@
 
 namespace flutter {
 
-// An adapter to receive DlCanvas calls and dispatch them into
-// an SkCanvas.
+// -----------------------------------------------------------------------------
+/// @brief      A Canvas implementation that delegates to an Skia-backed Canvas.
+///
+/// This class is intended to be used an internal implementation detail for the
+/// DisplayList package. @see DlSkCanvasDispatcher for the developer-facing
+/// interface to Skia.
```

Comment:
Whoops, I just noticed this is backwards.

DlCanvas is the public facing API.
DlOpReceiver is internal.

We want the framework, layers, embedders, other engine code to all be using the DlCanvas API. This class implements DlCanvas for Skia. While this class isn't itself the public facing API, it is implementing the public-facing API.

Reply:
Simplified to point at the interface.